egui/src/widgets/text_edit.rs

```diff
@@ -478,18 +523,24 @@ impl<'t> TextEdit<'t> {
                 let did_mutate_text = match event {
                     Event::Copy => {
                         if cursorp.is_empty() {
-                            copy_if_not_password(ui, text.clone());
+                            copy_if_not_password(ui, text.as_ref().to_owned());
```

Comment:
```suggestion
                            copy_if_not_password(ui, text.to_string());
```

is a bit shorter (and iirc it should work since `text` implements `Display`)

Reply:
Wouldn't calling `to_owned` on a `str` bypass the `std::fmt` formatting, though?
I don't know how it works in general, but there might be a specialization for `String` and `str` so it might not matter.